src/test/java/io/confluent/idesidecar/restapi/messageviewer/SimpleConsumerSuite.java

```diff
@@ -270,8 +270,7 @@ default void testProduceAndConsumeWithSnappyCompression() {
   }
 
   @ParameterizedTest
-  // TODO: Add "snappy" once we support it (see https://github.com/confluentinc/ide-sidecar/issues/304)
-  @ValueSource(strings = {"gzip", "lz4", "zstd"})
+  @ValueSource(strings = {"gzip", "lz4", "zstd", "snappy"})
```

Comment:
Make sure we can consume records that were compressed with snappy. Would love to run this in an integration test against the native executable on macOS and Windows once https://github.com/confluentinc/ide-sidecar/issues/429 is resolved.